src/components/Dropdown/components/ValueContainer/ValueContainer.jsx

```diff
@@ -10,12 +10,12 @@ import classes from "./ValueContainer.module.scss";
 const EMPTY_ARRAY = [];
 
 export default function Container({ children, selectProps, ...otherProps }) {
-  const { selectedOptions, onSelectedDelete, setIsDialogShown, isDialogShown, isMultiline } = selectProps.selectProps;
+  const { placeholder, selectProps: customProps = {} } = selectProps;
```

Comment:
`selectProps.selectProps`? 🤔

Reply:
that is the api 😅

Reply:
this holds all the props i pass as custom to the drop down sub components